pkg/utils/utils.go

```diff
@@ -38,3 +40,23 @@ func FileExists(path string) (bool, error) {
 
 	return false, err
 }
+
+// CheckHeptioAuthenticatorAWS checks for the authenticator binary existence.
+// DEPRECATED: Use CheckAuthenticatorBinary
+func CheckHeptioAuthenticatorAWS() error {
+	return CheckAuthenticatorBinary()
+}
+
+// CheckAuthenticatorBinary checks for the authenticator binary existence.
+func CheckAuthenticatorBinary() error {
```

Comment:
Simply calling it `CheckAuthenticator` (or indeed `checkAuthenticator`) would be fine. It's not a binary we are looking for, we want to see if command is in the path or not, binary maybe present anywhere else despite the fact...